### PR TITLE
BITMAKER-860: Modify kafka producers to send the jid

### PR DESCRIPTION
**Pull request by rodp63**
#### Ticket url(s):
- https://tasks.bitmaker.dev/issues/860

#### Description:
- Modify the middlewares and extensions to send the jid

#### Checklist:
- [X] Check that only one commit is added in the PR with the correct format
- [X] Rebase my branch

### CODE REVIEW

File: bm_scrapy/extensions.py [hunk 1]
```diff
@@ -29,9 +20,19 @@ def from_crawler(cls, crawler):
 
     def item_scraped(self, item):
         item = self.exporter.export_item(item)
-        self.producer.send('spider-items', value=dict(item))
+        data = {
+            'jid': os.getenv('BM_JOB'),
```
Comment (renato262) [inline bm_scrapy/extensions.py]: Maybe we should explain BM_JOB in the README.md, and change the name to a someting less general, DB_SPIDER_JOB. Maybe also KAFKA_ADVERTISED_HOST_NAME and KAFKA_ADVERTISED_PORT.

File: bm_scrapy/middlewares.py [hunk 2]
```diff
@@ -1,19 +1,28 @@
...
+                'status': int(response.status),
+                'method': request.method,
+                'duration': int(request.meta.get('download_latency', 0) * 1000),
+                'time': parse_time(),
+                'response_size': len(response.body),
+                'fingerprint': request_fingerprint(request),
+            },
+        }
+        self.producer.send('job_requests', value=data)
+        self.producer.flush()
```
Comment (renato262) [inline bm_scrapy/middlewares.py]: @rodp63  not so sure but `flush` is for sync call, maybe we should start to do send data asynchronously in this or other ticket

Comment (rodp63) [reply bm_scrapy/middlewares.py]: Yes, it will be approached in: https://tasks.bitmaker.dev/issues/880

File: bm_scrapy/extensions.py (hunk 1 above)
Comment (rodp63) [reply bm_scrapy/extensions.py]: Done

File: README.md [hunk 3]
```diff
@@ -10,3 +10,13 @@ The package implements a wrapper layer to extract job data from environment, pre
 ```
 python setup.py install 
 ```
+
+## Environmental variables
```
Comment (renato262) [inline README.md]: `Environment variables` is ok

**Pull request merged**